src/qml/components/Footer.qml

```diff
@@ -5,12 +5,13 @@ import QtQuick.Controls 2.12
 import "../theme"
 
 Item {
+    property string _text: ""
     height: 40
     width: parent.width
 
     Text {
         anchors.centerIn: parent
-        text: qsTr("Powered by Virgil Security, Inc.")
+        text: _text
         font.letterSpacing: 0.23
```

Comment:
Magic value. Define a named constant above for this, like tightLetterSpacing

Reply:
Done